_vendor/normalize-scss/sass/normalize/_normalize-mixin.scss

```diff
@@ -424,7 +424,7 @@
     optgroup,
     select,
     textarea {
-      font-family: inherit; /* 1 */
+      font-family: $base-font-family; /* 1 */
```

Comment:
Hi @LaurelBCodin This is not needed 
=> https://github.com/zurb/foundation-sites/pull/9973/files
Already added!